pootle/apps/pootle_store/migrations/0006_auto_20200610_2137.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 3.0.5 on 2020-06-10 21:37
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("pootle_store", "0005_auto_20200124_0617"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql=[
+                (
+                    """
+                CREATE PROCEDURE `insert_unique_submission` ( IN 
```

Comment:
I'd rename the procedure to `insert_submission`. The fact that it's unique shouldn't slip into the name (we will — hopefully! — never insert non-unique submission records).

Reply:
Agree